packages/nextjs/components/SwitchTheme.tsx

```diff
@@ -16,19 +14,10 @@ export const SwitchTheme = ({ className }: { className?: string }) => {
 
   return (
     <div className={`flex space-x-2 text-sm ${className}`}>
-      <input
-        id="theme-toggle"
-        type="checkbox"
-        className="toggle toggle-primary bg-primary hover:bg-primary border-primary"
-        onChange={toggle}
-        checked={isDarkMode}
-      />
-      {isMounted() && (
-        <label htmlFor="theme-toggle" className={`swap swap-rotate ${!isDarkMode ? "swap-active" : ""}`}>
-          <SunIcon className="swap-on h-5 w-5" />
-          <MoonIcon className="swap-off h-5 w-5" />
-        </label>
-      )}
+      <label onClick={toggle} htmlFor="theme-toggle" className={`swap swap-rotate ${!isDarkMode ? "swap-active" : ""}`}>
+        <SunIcon className="swap-on h-5 w-5" />
+        <MoonIcon className="swap-off h-5 w-5" />
```

Comment:
These should be switched. When light mode is activated I want to see a moon icon. When dark mode is activated I want to see a sun icon. This way the user knows they can click the icon to switch.